ceph/rados/core_workflows.py

```diff
@@ -986,15 +986,6 @@ def verify_reweight(self, affected_osds: list, osd_info: list) -> bool:
 
         return True
 
-    def list_pools_on_cluster(self) -> list:
```

Comment:
Any reason why we are removing this module?

Reply:
Ignore the above comment, going through the code I realized we already have a module `list_pools` which does the same thing

Reply:
We already had a module list_pools(), that's why this was removed.